examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3659,6 +3751,235 @@ class GLTFParser {
+						case 'emissiveFactor':
+							targetProperty = 'emissive';
+							break;
+						case 'alphaCutoff':
+							targetProperty = 'alphaTest';
+							break;
+						case 'extensions/KHR_materials_emissive_strength/emissiveStrength':
+							targetProperty = 'emissiveIntensity';
+							break;
+						case 'baseColorTexture/extensions/KHR_texture_transform/scale':
+							targetProperty = 'map/repeat';
+							break;
+						case 'baseColorTexture/extensions/KHR_texture_transform/offset':
+							targetProperty = 'map/offset';
+							break;
+						case 'emissiveTexture/extensions/KHR_texture_transform/scale':
+							targetProperty = 'emissiveMap/repeat';
+							break;
+						case 'emissiveTexture/extensions/KHR_texture_transform/offset':
+							targetProperty = 'emissiveMap/offset';
+							break;
+
+					}
+
+					path = pathStart + targetProperty;
+					if ( animationPointerDebug ) console.log( 'PROPERTY PATH', pathStart, targetProperty, path );
+					break;
+
+				case TARGET_TYPE.node:
+					const pathIndexNode = ( '/nodes/' + targetId.toString() + '/' ).length;
+					const pathStartNode = path.substring( 0, pathIndexNode );
+					targetProperty = path.substring( pathIndexNode );
+
+					switch ( targetProperty ) {
+
+						case 'translation':
+							targetProperty = 'position';
+							break;
+						case 'rotation':
+							targetProperty = 'quaternion';
+							break;
+						case 'scale':
+							targetProperty = 'scale';
+							break;
+						case 'weights':
+							targetProperty = 'morphTargetInfluences';
+							break;
+
+						// TODO matrix
+
+					}
+
+					path = pathStartNode + targetProperty;
+					break;
+
+				case TARGET_TYPE.light:
+					const pathIndexLight = ( '/extensions/KHR_lights_punctual/lights/' + targetId.toString() + '/' ).length;
+					// const pathStartLight = path.substring( 0, pathIndexLight );
+					targetProperty = path.substring( pathIndexLight );
+
+					switch ( targetProperty ) {
+
+						case 'color':
+							break;
+						case 'intensity':
+							break;
+						case 'spot/innerConeAngle':
+							// TODO need to set .penumbra, but requires calculations on every animation change (?)
+							targetProperty = 'penumbra';
+							break;
+						case 'spot/outerConeAngle':
+							targetProperty = 'angle';
+							break;
+						case 'range':
+							targetProperty = 'distance';
+							break;
+
+					}
+
+					path = '/lights/' + targetId.toString() + '/' + targetProperty;
+					break;
+
+				case TARGET_TYPE.camera:
+					const pathIndexCamera = ( '/cameras/' + targetId.toString() + '/' ).length;
+					const pathStartCamera = path.substring( 0, pathIndexCamera );
+					targetProperty = path.substring( pathIndexCamera );
+
+					switch ( targetProperty ) {
+
+						case 'yfov':
+							targetProperty = 'fov';
+							break;
+						case 'znear':
+							targetProperty = 'near';
+							break;
+						case 'zfar':
+							targetProperty = 'far';
+							break;
+						case 'aspect':
+							break;
```

Comment:
Similar to above, will have to call `camera.updateProjectionMatrix()` after any of these changes.

I'm not sure what I think about animation of the aspect ratio. glTF lacks any meaningful way to express "let the camera inherit the aspect ratio from the application", animating this property kind of seems like stacking blogs on top of something that is already not working properly.